app/src/main/java/com/ahugenb/hra/calculator/CalculatorViewModel.kt

```diff
@@ -0,0 +1,37 @@
+package com.ahugenb.hra.calculator
+
+import androidx.lifecycle.ViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+class CalculatorViewModel: ViewModel() {
+    companion object {
+        const val OZ_ETHANOL_TO_UNITS: Double = 10.0 / 6.0
+        const val ML_ETHANOL_TO_UNITS: Double = 1.0 / 17.7
+        const val PERCENT: Double = 100.0
+    }
+
+    private val _units: MutableStateFlow<Double> = MutableStateFlow(0.0)
+    val units: StateFlow<Double> = _units
```

Comment:
Looks cool! This is kind of what I was talking about earlier... Instead of using 2 `StateFlow`s, you could emit a data class containing both values.
```
data class CalculatorState(val units: Double = 0.0, val pureEthanol Double = 0.0){}

val viewState: StateFlow<CalculatorState> = MutableStateFlow(CalculatorState())
```